karton/misp_pusher/misp_pusher.py

```diff
@@ -113,6 +125,6 @@ def main(cls):
 
         config = Config(args.config_file)
         service = cls(
-            config, args.misp_url, args.misp_key, not args.misp_insecure, args.mwdb_url
+            config, args.misp_url, args.misp_key, args.misp_published, not args.misp_insecure, args.mwdb_url
```

Comment:
To make style chcker hapy

```suggestion
            config,
            args.misp_url,
            args.misp_key,
            args.misp_published,
            not args.misp_insecure,
            args.mwdb_url,
```